Veil-Framework Version 3.1.6 Installation Issues

## Veil version
      - 3.1.6

## OS Used - all info (architecture, linux flavor, etc)
     -Distributor ID: Kali
     -Description:    Kali GNU/Linux Rolling
     -Release:        2022.2
     -Codename:       kali-rolling
     - Architecture : x64 

## How did you install Veil? (Apt, Clone from Github, etc.)
     - Download the zip file and I run command ./setup.sh to install


## Pastebin link to error you are encountering (include console actions you took prior to error)
     
Receive error as follows:
===================

Collecting pefile
  Downloading https://files.pythonhosted.org/packages/f9/1e/fc4fac0169d16a98577809400bbcfac8ad1900fa792184327b360ea51fc6/pefile-2021.5.13.tar.gz (66kB)
     |████████████████████████████████| 71kB 1.1MB/s
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):                                                                                                                         
      File "<string>", line 1, in <module>                                                                                                                            
      File "C:\users\kali\Temp\pip-install-zfgvavns\pefile\setup.py", line 86, in <module>                                                                            
        long_description = "\n".join(_read_doc().split('\n')),                                                                                                        
      File "C:\users\kali\Temp\pip-install-zfgvavns\pefile\setup.py", line 33, in _read_doc                                                                           
        tree = ast.parse(f.read())                                                                                                                                    
      File "C:\Python34\lib\ast.py", line 35, in parse                                                                                                                
        return compile(source, filename, mode, PyCF_ONLY_AST)                                                                                                         
      File "<unknown>", line 3789                                                                                                                                     
        f'Export directory contains more than 10 repeated entries '                                                                                                   
                                                                  ^                                                                                                   
    SyntaxError: invalid syntax                                                                                                                                       
    ----------------------------------------                                                                                                                          
ERROR: Command "python setup.py egg_info" failed with error code 1 in C:\users\kali\Temp\pip-install-zfgvavns\pefile\                                                 
 [ERROR] Failed to run (wine) Python pip pefile... Exit code: 1        

## Any additional info you want to tell me
    - I also received error "If you have any errors running Veil, run: './Veil.py --setup' and select the nuke the wine folder option"
    - then I run command  "./Veil.py --setup"
    - receive output as follows: 

     "ERROR #2-3: Can't find the WINE profile for AuotIT v3 (/var/lib/veil/wine//drive_c/Program Files/AutoIt3/Aut2Exe/Aut2exe.exe).   
       Run: /home/kali/Downloads/Veil-master/config/setup.sh --force --silent"

Anyone here can help me to resolve error as above..?

Thanks

--- a/config/setup.sh
+++ b/config/setup.sh
@@ -811,7 +811,7 @@ func_python_deps(){
     echo -e " ${RED}[ERROR] ${msg}${RESET}\n"
   fi
 
-  sudo -u "${trueuser}" WINEPREFIX="${winedir}" wine "${winedir}/drive_c/Python34/python.exe" "-m" "pip" "install" "pefile"
+  sudo -u "${trueuser}" WINEPREFIX="${winedir}" wine "${winedir}/drive_c/Python34/python.exe" "-m" "pip" "install" "-Iv" "pefile==2019.4.18"
   tmp="$?"
   if [[ "${tmp}" -ne "0" ]]; then
     msg="Failed to run (wine) Python pip pefile... Exit code: ${tmp}"

